OasisDEX/oasis-borrow
# Informational message is always visible -> should only be visible on changes in the slider

# [Informational message is always visible -> should only be visible on changes in the slider](https://app.shortcut.com/oazo-apps/story/6489/informational-message-is-always-visible-should-only-be-visible-on-changes-in-the-slider)

## Changes 👷‍♀️
  <Please add short list of changes>
- only shows message on adjust when there is pending transaction data
- fixes the reset link on open and manage
- refreshes page on button press at end of adjust flow (need to get new chain data)
- shows total total percentage until liquidation rather than topping out at 20%
  
## How to test 🧪
  <Please explain how to test your changes>
- open flow - should always show message and transaction data, even when resetting
- manage - should only show transaction data and message when we have adjusted risk away from current on-chain position
- should refresh page on button click at end of adjust flow
